src/index.js

```diff
@@ -82,7 +83,16 @@ function configureInterceptors(instance, url, { username, password } = {}) {
 
   instance.interceptors.response.use(res => {
     return res.data;
+  }, err => {
+    if (!(err instanceof Error) && typeof err === "object") {
+      return Promise.reject(errorify(err));
+    }
+    return Promise.reject(err);
```

Comment:
Cool, I'll update it.  Also the test are munged because it needs to connect to a live auth server.  I'm going to open an issue to mock auth for the tests so it doesn't need an extra service to run.